manageShips.php

```diff
@@ -0,0 +1,74 @@
+<?php
+require __DIR__.'/bootstrap.php';
+// Page to manage and display a ship table and link to view ships alone
+// for Edit/ Delete or Add ships
+
+use Service\Container;
+
+$container = new Container($configuration);
+
+$shipLoader = $container->getShipLoader();
+$ships = $shipLoader->getShips();
+?>
+
+<html>
```

Comment:
I think we can do the "messy" include header here and move everything from the `<html>` to the `<body>` tag into that header file. And do the same for the footer. So we can just include it on each page instead of having to duplicate that every page.

Reply:
Header and footer layout files created